seed/factories/user.factory.ts

```diff
@@ -16,7 +16,7 @@ export const userFactory = generateFactory<User>(() => {
     name,
     email: faker.internet.email(),
     password: generateRandomPassword(),
-    phone: faker.phone.number({ style: 'national' }),
+    phone: faker.helpers.fromRegExp(/[0-9]{10}/),
```

Comment:
Let's use the numeric string helper here too:
```suggestion
    phone: faker.string.numeric({ length: 10, allowLeadingZeros: false }),
```

We should prolly find a way to centralize faker field definitions like this as well.

Maybe w/ something like https://www.npmjs.com/package/@anatine/zod-mock ? 🤔